UnityProject/Assets/Scripts/Items/Cargo/Wrapping/WrappedBase.cs

```diff
@@ -24,25 +26,22 @@ public abstract class WrappedBase: MonoBehaviour
 		[SerializeField] [Tooltip("Time to unwrap.")]
 		private float timeToUnwrap = 2;
 
-		private PushPull content;
-		private PushPull pushPull;
 		private Attributes attributes;
 		protected SpriteHandler spriteHandler;
 
-		protected virtual void Awake()
+		protected virtual void OnEnable()
 		{
-			pushPull = GetComponent<PushPull>();
 			spriteHandler = GetComponentInChildren<SpriteHandler>();
 			attributes = GetComponent<Attributes>();
 		}
 
 		public void SetContent(GameObject toWrap)
 		{
-			content = toWrap.GetComponent<PushPull>();
+			StoreObject(toWrap);
 			var exportCost = toWrap.GetComponent<Attributes>().ExportCost;
 			UpdateExportCost(exportCost);
-			content.parentContainer = pushPull;
-			content.VisibleState = false;
+			toWrap.GetComponent<PushPull>().VisibleState = false;
```

Comment:
I have an uneducated question, since this is now using ObjectContainer to handle *containing* objects inside another, why is it needed to manually move the content to hidden zone and assign the container as parent? Doesn't ObjectContnainer already do that?

Reply:
You are correct, that's a good catch. This is something that escaped my cleanup/review. Just a paranoid holdover from poking and prodding.